livre/core/render/Frustum.h

```diff
@@ -36,7 +36,15 @@ namespace livre
 class Frustum : public Frustumf
 {
 public:
-    LIVRECORE_API Frustum();
+
+    /**
+     * Initializes the Frustum parameters from the modelview and projection matrices.
+     * @param modelViewMatrix 4x4 modelview matrix.
+     * @param projectionMatrix 4x4 projection matrix,
+     */
+    LIVRECORE_API Frustum( const Matrix4f& modelViewMatrix = Matrix4f::IDENTITY,
+                           const Matrix4f& projectionMatrix = Matrix4f::IDENTITY );
```

Comment:
I thought about that but and empty frustum is an empty frustum. It wont change anything when multiplied with the including matrices. That is why I did it. But I can do that way as well.